File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/15/15.js
// Q15: Print binary alternating pattern (1, 01, 101, 0101).
// Input: An integer n
// Output: Binary alternating pattern

const readline = require('readline');
const rl = readline.createInterface({ input: process.stdin });
rl.on('line', (line) => {
    const n = parseInt(line);
    for (let i = 1; i <= n; i++) {
        let row = "";
        for (let j = 0; j < i; j++) {
            row += (i + j) % 2 === 0 ? "1" : "0";
        }
        console.log(row);
    }
    rl.close();
});

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/15/15.py
# Q15: Print binary alternating pattern (1, 01, 101, 0101).
# Input: An integer n
# Output: Binary alternating pattern

n = int(input())
for i in range(1, n + 1):
    row = ""
    for j in range(i):
        if (i + j) % 2 == 0:
            row += "1"
        else:
            row += "0"
    print(row)
